src/EditorFeatures/Core/Remote/SolutionChecksumUpdater.cs

```diff
@@ -60,9 +60,6 @@ internal sealed class SolutionChecksumUpdater
             listener,
             shutdownToken);
 
-        // Use an equality comparer here as we will commonly get lots of change notifications that will all be
-        // associated with the same cancellation token controlling that batch of work.  No need to enqueue the same
-        // token a huge number of times when we only need the single value of it when doing the work.
```

Comment:
comment was stale.  no equality comaprer was passed at all.  this was before we had the arg-less ABWQ.